openlibrary/plugins/openlibrary/js/partner_ol_lib.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @param {string} container
+ */
+function getIsbnToElementMap(container) {
+    const reISBN = /((978)?[0-9][0-9]{10}[0-9xX])|((978)?[0-9]{9}[0-9Xx])/;
+    const elements = Array.from(document.querySelectorAll(container));
+    const isbnElementMap = {};
+    elements.forEach((e) => {
+        const isbnMatches = e.innerHTML.match(reISBN);
```

Comment:
This will let us match cases like `<a href="http://blah.org/97812312312">`
```suggestion
        const isbnMatches = e.outerHTML.match(reISBN);
```